src/serviceorchestration/5_0_0/sd/orchestration/orchestration_sd.tex

```diff
@@ -0,0 +1,441 @@
+
+    If \texttt{false}, orchestration process returns all the matching service instance if any. 
+    
+    \item \texttt{ONLY\_PREFERRED}:
+
+    If \texttt{true}, orchestration process considers only those matching service instances that are provided by a preferred provider if any.
+
+    If \texttt{false}, but preferred providers are specified and have matching service instances, than orchestration process considers only those service instances that are provided by a preferred provider. Otherwise, non-preferred providers are considered.
+    
+    \item \texttt{ONLY\_EXCLUSIVE}:
+
+    If \texttt{true}, orchestration process considers only those matching service instances that are allows exclusivity. It automatically results \texttt{MATCHMAKING:true} as well.
+
+    If \texttt{false}, but exclusivity duration is specified and there are matching services with exclusivity allowed, than orchestration process considers only those service instances that allows exclusivity. Otherwise, service instances without exclusivity are considered.
+    
+    \item \texttt{ALLOW\_INTERCLOUD}:
+
+    If \texttt{true}, orchestration process considers matching service instances from neighbor clouds when there are no local hits. Orchestrating from neighbor clouds automatically results \texttt{MATCHMAKING:true}
+
+    If \texttt{false}, orchestration process doesn't consider matching service instances from neighbor clouds when there are no local hits.
+
+    \item \texttt{ONLY\_INTERCLOUD}:
+
+    If \texttt{true}, orchestration process considers matching service instances only from the neighbor clouds. It automatically results \texttt{MATCHMAKING:true} as well.
+
+    If \texttt{false}, orchestration process considers matching service instances from the local could in first hand and only considers matching service instances from the neighbor clouds when \texttt{ALLOW\_INTERCLOUD:true}.
+    
+    \item \texttt{ALLOW\_TRANSLATION}:
+
+    If \texttt{true}, orchestration process considers matching, but non-native service instances when there are no native hits. It automatically results \texttt{MATCHMAKING:true} as well.
+
+     If \texttt{false}, orchestration process considers only native matching service instances.
+    
+\end{itemize}
+
+\msubsection{struct}{QoSRequirementMap}
+
+An \pref{Object} which maps \pref{String} keys \pref{String} values.
+
+\msubsection{struct}{OrchestrationResponse}
+ 
+\begin{table}[ht!]
+\begin{tabularx}{\textwidth}{| p{3cm} | p{4.25cm} | X |} \hline
+\rowcolor{gray!33} Field & Type & Description \\ \hline
+results & \pref{List}$<$\hyperref[sec:model:OrchestrationResult]{OrchestrationResult}$>$ & List of matching service instances. \\ \hline
+warnings & \pref{List}$<$\hyperref[sec:model:String]{String}$>$ & Warning message list. \\ \hline
+\end{tabularx}
+\end{table}
+
+\msubsection{struct}{OrchestrationResult}
```

Comment:
Somewhere here we should mention that in case of simple-store only the serviceInstanceId is filled.